website/docs/comparisons/s2.md

```diff
@@ -0,0 +1,38 @@
+---
+id: s2
+title: S2
+sidebar_label: S2
+slug: /comparisons/s2
+---
+
+[S2](https://s2geometry.io/), like H3, implements an open source, hierarchical, discrete, and global grid system. The systems share a number of similarities, including the use of 64 bit integers as cell indexes, making it very efficient to use both of them in big data systems.
+
+One of the main differences between S2 and H3 is the choice of cell shape. S2 uses square cells while H3 uses hexagonal cells. This leads to several important differences.
```

Comment:
I think we're missing space-filling: "Hexagons are also optimally space-filling, i.e. on average a polygon may be filled with hexagon tiles with a smaller margin of error than would be present with square tiles."